src/schema/v2/me/sendIdentityVerficationEmailMutation.ts

```diff
@@ -41,6 +41,12 @@ const IdentityVerificationEmailType = new GraphQLObjectType<
       type: GraphQLString,
       resolve: ({ user_id }) => user_id,
     },
+    verificationURL: {
```

Comment:
It seems a bit odd to me to include this as part of the `Email` type (and mutation). That sort of ties a schema choice to a specific UI of the day (wanting to render this link after creation).

Instead, this looks like it's a property/field that maybe belongs on the `IdentityVerification` type itself?

Reply:
@mzikherman The suggested change makes sense to me but I am struggling with making the connection as far as how this affects the response type.

If the field is (instead) added to `IdentityVerification` type, does the [`IdentityVerificationEmailMutationSuccessType`](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/me/sendIdentityVerficationEmailMutation.ts#L51-L59) then need to be updated and return that type as an additional field? 

```
  fields: () => ({
    identityVerificationEmail: {
      type: IdentityVerificationEmailType,
      resolve: (identityVerification) => identityVerification,
    },
    identityVerification: {
      type: IdentityVerificationType,
      resolve: (identityVerification) => identityVerification,
    },
```

Reply:
ty @mzikherman ! same Q as @ovasdi

Reply:
@mzikherman to further clarify what you suggest:

Add something like the following to [`IdentityVerificationType`](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/identityVerification.ts#L134)?:
```
    verificationURL: {
      description:
        "Verification page URL sent to the identify verification's owner",
      type: GraphQLString,
      resolve: (res) => {
        return `${config.FORCE_URL}/identity-verification/${res.id}`
      },
    },
```

then (perhaps) add a field to SuccessType:
```
  name: "IdentityVerificationEmailMutationSuccessType",
  isTypeOf: (data) => data.id,
  fields: () => ({
    identityVerificationEmail: {
      type: IdentityVerificationEmailType,
      resolve: (identityVerification) => identityVerification,
    },
    verificationURL: {
      type: GraphQLString,
      resolve: (identityVerification) => identityVerification.verificationURL,
    },
  }),
```

Then update [`mutateAndGetPayload`](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/me/sendIdentityVerficationEmailMutation.ts#L121-L122) adding an `identityVerificationLoader` to fetch the data so we can extract the URL?